### ISSUE TITLE
Decide what to do with grid-acq model box warnings

### ISSUE THREAD

**Issue by jeanconn** [sot/starcheck#268]
```
Checking star catalog for obsid 48613
/proj/sot/ska3/flight/lib/python3.6/site-packages/chandra_aca/star_probs.py:297: UserWarning: 
Model grid-floor-2018-11 computed between halfw <= 60.0 <= 180.0, clipping input halfw(s) outside that range.
  .format(model, name, val_lo, val_hi, name))
```

**Comment by taldcroft** [comment]
Doing nothing is an option and is perfectly safe (more conservative) and when proseco is in place this is extremely unlikely to recur.

But if you really want to do something, then clipping to 60 and putting in a green warning would be fine.  Based on maneuver error statistics I do not believe we should ever be using boxes smaller than 60, so that's fine as a green warning if one shows up in a catalog.

**Comment by jeanconn** [comment]
I don't think I have to "do" the clipping to 60, because I thought from that warning it is already done.  Just wondering if there's any value to bringing it to the ACA reviewer's attention that we're out of range.  I don't think so, but also figure you put the UserWarning in for a reason.

Writing it out to standard out/err probably has no value; doesn't hurt anything either (but we've generally tried not to put anything to console unless it actually has value).

**Comment by taldcroft** [comment]
I meant that you check for less than 60, put in a warning, and then clip to 60 in order to get rid of that warning to console.  We definitely do not want to see warnings like that in normal runs.

**Comment by jeanconn** [comment]
OK.  If the limits (60 to 180) are unlikely to change, that seems fine.

**Comment by jeanconn** [comment]
This will be resolved by #272 which warns on halfwidths out of the range and passes the rest to proseco (which results in no warning to console).
